src/main/java/org/eocqrs/kafka/Consumer.java

```diff
@@ -28,6 +28,6 @@
  */
 public interface Consumer<X> {
 
-  void consume(int partition, Data<X> message);
+  Dataized<?> consume(int partition, Data<X> message);
```

Comment:
1. To consume data we need partition and other meta information
2. I would name this method not consume, but dataized[read [this](https://www.yegor256.com/2018/08/22/builders-and-manipulators.html)]
3. finally, maybe we need a puzzle

what do you think @l3r8yJ